pennylane/workflow/_capture_qnode.py

```diff
@@ -13,6 +13,98 @@
+jaxpr is directly sent to the device, whether or not the device can handle it.
+
+>>> @qml.qnode(qml.device('default.qubit', wires=3))
+... def circuit():
+...     qml.Permute(jax.numpy.array((0,1,2)), wires=(2,1,0))
+...     return qml.state()
+>>> circuit()
+MatrixUndefinedError:
+
+*Transforms are still under developement*.  No transforms will currently be applied as part of the workflow.
+
+*Breaking vmap/ parameter broadcasting into a non-broadcasted state*. The current workflow assumes
+that the device execution can natively handled broadcasted parameters. vmap and parameter broadcasting
+will not work with devices other than default qubit.
+
+>>> @qml.qnode(qml.device('lightning.qubit', wires=1))
+... def circuit(x):
+...     qml.RX(x, 0)
+...     return qml.expval(qml.Z(0))
+>>> jax.vmap(circuit)(jax.numpy.array([1.0, 2.0, 3.0]))
+TypeError: RX(): incompatible function arguments. The following argument types are supported:
+    1. (self: pennylane_lightning.lightning_qubit_ops.StateVectorC128, arg0: list[int], arg1: bool, arg2: list[float]) -> None
+    2. (self: pennylane_lightning.lightning_qubit_ops.StateVectorC128, arg0: list[int], arg1: list[bool], arg2: list[int], arg3: bool, arg4: list[float]) -> None
+
+*Grouping commuting measurements and/ or splitting up non-commuting measurements.* Currently, each
+measurment is fully independent and generated from different raw samples than every other measurement.
+To generate multiple measurments from the same samples, we need a way of denoting which measurements
+should be taken together.  A "Combination measurement process" higher order primitive, or something like it.
+We will also need to figure out how to implement splitting up a circuit with non-commuting measuremets into
+multiple circuits.
+
+>>> @qml.qnode(qml.device('default.qubit', wires=1, shots=5))
+... def circuit():
+...     qml.H(0)
+...     return qml.sample(wires=0), qml.sample(wires=0)
+>>> circuit()
+(Array([1, 0, 1, 0, 0], dtype=int64), Array([0, 0, 1, 0, 0], dtype=int64))
+
+*Figuring out what types of data can be sent to the device.* Is the device always
+responsible for converting jax arrays to numpy arrays? Is the device responsible for having a
+pure-callback boundary if the execution is not jittable? We do have an opportunity here
+to have gpu-end-to-end simulation on lightning gpu and lightning kokkos.
+
+*Jitting workflows involving qnodes*. While the execution of jaxpr on default qubit is
+currently jittable, we will need to register a lowering for the qnode primitive.  We will also
+need to figure out where to apply a ``jax.pure_callback`` for devices like lightning qubit that are
+not jittable.
+
+*Result caching*. The new workflow is not capable of caching the results of executions, and we have
+not even started thinking about how it might be possible to do so.
+
+*Unknown other features*. The workflow currently has limited testing, so this list of unsupported
+features is non-exhaustive.
```

Comment:
```suggestion
**Jitting workflows involving qnodes**. While the execution of jaxpr on ``default.qubit`` is
currently jittable, we will need to register a lowering for the qnode primitive.  We will also
need to figure out where to apply a ``jax.pure_callback`` for devices like ``lightning.qubit`` that are
not jittable.

**Result caching**. The new workflow is not capable of caching the results of executions, and we have
not even started thinking about how it might be possible to do so.

**Unknown other features**. The workflow currently has limited testing, so this list of unsupported
features is non-exhaustive.
```